improver_tests/ensemble_copula_coupling/test_EnsembleReordering.py

```diff
@@ -297,21 +297,38 @@ def test_2d_cube_tied_values(self):
         matches = [np.array_equal(aresult, result.data) for aresult in permutations]
         self.assertIn(True, matches)
 
-    def test_2d_cube_tied_values_random_seed(self):
+    def test_2d_cube_tied_values_random_with_seed(self):
         """
         Test that the plugin returns the correct cube data for a
-        2d input cube, when there are tied values witin the
+        2d input cube, when there are tied values within the
         raw ensemble realizations. The random seed is specified to ensure that
         only one option, out of the two possible options will be returned.
         """
         raw_data = np.array([[1, 1], [3, 2], [2, 2]])
         calibrated_data = np.array([[1, 1], [2, 2], [3, 3]])
+        result_data = np.array([[1, 1], [3, 3], [2, 2]])
+
+        raw_cube = self.cube_2d.copy(data=raw_data)
+        calibrated_cube = self.cube_2d.copy(data=calibrated_data)
+
+        result = Plugin().rank_ecc(calibrated_cube, raw_cube, random_seed=1)
+        self.assertArrayAlmostEqual(result.data, result_data)
```

Comment:
Any particular reason this test was modified? Is it simply to differentiate it to the test below?

Reply:
Yes, I modified the test so that `result_data` was different for the 2 tests. My aim was to demonstrate the difference between using random and realisation tie-breaking.